packages/near-fast-auth-signer/src/utils/multi-chain/signature/signature.ts

```diff
@@ -29,28 +29,18 @@ const toRVS = (signature: string): RSVSignature => {
  * @param {string | ethers.BytesLike} transactionHash - The hash of the transaction to be signed.
  * @param {string} path - The derivation path used for signing the transaction.
  * @param {NearAuthentication} nearAuthentication - The NEAR accountId, keypair and networkId used for signing the transaction.
- * @param {string} relayerUrl - The URL of the relayer service to which the signed transaction is sent.
  * @param {ChainSignatureContracts} contract - The contract identifier for chain signature operations.
+ * @param {string} relayerUrl - The URL of the relayer service to which the signed transaction is sent.
  * @returns {Promise<RSVSignature>} A promise that resolves to the RSV signature of the signed transaction.
  * @throws {Error} Throws an error if the signature cannot be fetched after 3 attempts.
  */
 export const sign = async (
   transactionHash: string | ethers.BytesLike,
   path: string,
   nearAuthentication: NearAuthentication,
-  relayerUrl: string,
-  contract: ChainSignatureContracts
+  contract: ChainSignatureContracts,
+  relayerUrl?: string
```

Comment:
since sign function now accepts 5 arguments, would be ideal to change the argument with object with props. In that way order doesn't really matter and reduces chance of making mistake.